Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.5-alpine as base
```

Comment:
Here, we need the `as base` because a few lines down below we refer to this first image to copy the documentation (`FROM base AS documentation`).
Give it a go I believe that's all it is needed 👍